messaginginapp/src/main/java/io/customer/messaginginapp/type/InAppEventListener.kt

```diff
@@ -4,5 +4,5 @@ interface InAppEventListener {
     fun messageShown(message: InAppMessage)
     fun messageDismissed(message: InAppMessage)
     fun errorWithMessage(message: InAppMessage)
-    fun messageActionTaken(message: InAppMessage, action: String, name: String)
+    fun messageActionTaken(message: InAppMessage, action_value: String, action_name: String)
```

Comment:
Just a question, can't we use camel case here? e.g. `actionValue` and `actionName`? 🤓

Reply:
Good call out, done